infra/release/release_note.md

```diff
@@ -1,28 +1,27 @@
-# Release Note 0.2.0
+# Release Note 0.3.0
 
 ## ONE-vscode
 
 ### ONE Explorer View
 
-- More commands are supported.
-  - [Open with Text Editor] command
-  - [Open Containing Folder] command
-  - [Collapse All] command
-- Improve [Create Config] command
+- TODO: Fill this part
```

Comment:
```suggestion
- Add hide-and-show button
- Show more files
  - memory trace files
  - performance trace files
  - circle log files
  - extra tv2 files
- Open files with dedicated editor/viewer
  - circle files - 'Circle Viewer'
  - memory trace files - 'Mondrian Viewer'
  - performance trace files - 'Json Tracer'
  - circle log files - 'Text Editor'

```

Reply:
Q) Is `tv2` necessary or acceptable to mention in release note? I'm thinking of `- backend model binary files` as alternative.

Reply:
I editted ;-D